mp/src/game/server/momentum/mom_timer.cpp

```diff
@@ -316,9 +316,9 @@ void CMomentumTimer::DisablePractice(CMomentumPlayer *pPlayer)
 CON_COMMAND(mom_start_mark_create, "Marks a starting point inside the start trigger for a more customized starting location.\n")
 {
     const auto pPlayer = CMomentumPlayer::GetLocalPlayer();
-    if (pPlayer)
+    if (pPlayer && pPlayer->CreateStartMark())
     {
-        pPlayer->CreateStartMark();
+        ClientPrint(pPlayer, HUD_PRINTTALK, "Start Mark Created!");
```

Comment:
These prints can be moved to the start mark functions. Better to keep them there than to have to do it everywhere start mark stuff is called.

Reply:
Mainly was doing this as the one for clear has to be suppressed since the create function calls it.

Reply:
I would make it a bool parameter then, like `bPrintMsg` that defaults to true but can be set to false for not printing it

Reply:
Yeah exactly